controls/scheduler/built-in-dialogs.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Built-in Dialogs
+page_title: .NET MAUI Scheduler Documentation - Built-in Dialogs
+description: Learn more about the Telerik .NET MAUI Scheduler built-in dialogs for creating, editing and deleting appointments.
+position: 9
+slug: scheduler-built-in dialogs
+---
+
+# .NET MAUI Scheduler Built-in Dialogs
+
+Telerik .NET MAUI Scheduler control exposes built-in dialogs for creation and modification of appointments, so you can provide users with the ability to directly schedule their meetings.
+
+Check below the available dialogs for managing appointments.
```

Comment:
Consider re-working the list into H4s. Right now the screenshots don't nest that well visually. 

For example:

```
#### Dialog Name

Description from the bullet.

![Screenshots](...)
```

Here's how that would look like, if you decide to go that way.

![image](https://github.com/telerik/maui-docs/assets/3539221/0fc4c041-8912-47a5-8afc-af4a68561eb9)

Reply:
Agree